extensions/src/AWSSDK.Extensions.Bedrock.MEAI/BedrockChatClient.cs

```diff
@@ -79,7 +84,34 @@ public async Task<ChatResponse> GetResponseAsync(
         request.InferenceConfig = CreateInferenceConfiguration(request.InferenceConfig, options);
         request.AdditionalModelRequestFields = CreateAdditionalModelRequestFields(request.AdditionalModelRequestFields, options);
 
-        var response = await _runtime.ConverseAsync(request, cancellationToken).ConfigureAwait(false);
+        // Execute the request with proper error handling for ResponseFormat scenarios
+        ConverseResponse response;
+        try
+        {
+            response = await _runtime.ConverseAsync(request, cancellationToken).ConfigureAwait(false);
+        }
+        catch (AmazonBedrockRuntimeException ex) when (options?.ResponseFormat is ChatResponseFormatJson)
```

Comment:
why are we checking bedrockruntimeexception here but down below i see we are throwing InvalidOperationException when it fails?